docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to gnmi-netconf-adapter
```

Comment:
If you want to provide pointers to the general contribution guidelines, I would suggest to do it in the README page. We don't duplicate contribution guidelines in every repo which can be confusing. The docs in each repo should keep only subsystem related docs. In case you want to have a document about the build process, I would suggest to rename it to build.md or prerequisite.md (lower case) (e.g. onos-gui).

Reply:
Thanks @adibrastegarnia for the comments.  I was planning that the doc contained general information relating to contributing to this project (including build information required by the developers), a catchall, that wasn't covered in the onos CONTRIBUTING guide.   Happy to follow your recommendations.